src/test/java/spoon/support/JavaOutputProcessorTest.java

```diff
@@ -0,0 +1,62 @@
+package spoon.support;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import spoon.Launcher;
+import spoon.reflect.declaration.*;
+import spoon.reflect.factory.Factory;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class JavaOutputProcessorTest {
+
+
+    @Test
+    void testCreateModuleFileAssertAnnotationFileCreated(@TempDir File tempDir) {
+        // contract : createModuleFile creates an annotation file for module and prints it
+
+        // arrange
+        Launcher launcher = new Launcher();
+        launcher.setSourceOutputDirectory(tempDir.getAbsolutePath());
+        launcher.getEnvironment().setComplianceLevel(9);
+        Factory factory = launcher.getFactory();
+
+        String moduleName = "emptyModule";
+        CtModule module = factory.Module().getOrCreate(moduleName);
+        JavaOutputProcessor javaOutputProcessor = new JavaOutputProcessor();
+        javaOutputProcessor.setFactory(factory);
+
+        // act
+        javaOutputProcessor.process(module);
+        File expectedFile = tempDir.toPath().resolve("emptyModule/module-info.java").toFile();
```

Comment:
This is really nitpicking, but in the AAA pattern, computing an expected value is not part of the `act` phase. It either belongs in `arrange` or `assert`, depending on who you ask. For a value as simple as this one, I'd definitely put it in the `assert` phase to keep the variable use close to its definition.